Gutenberg/cocoapods_helpers.rb

```diff
@@ -74,28 +76,31 @@ def gutenberg_dependencies(options:)
 
   computed_dependencies = DEPENDENCIES.dup
 
-  # Use a custom RNReanimated version while we coordinate a fix upstream
-  computed_dependencies.delete('RNReanimated')
-  pod 'RNReanimated', git: 'https://github.com/wordpress-mobile/react-native-reanimated', branch: 'mokagio/fix-custom-node_modules-bypass-multiple-versions-check-2.17.0'
+  react_native_version = react_native_version!(gutenberg_path: gutenberg_path)
+  # We need to apply a workaround for the RNReanimated library when using React Native 0.71+.
+  apply_rnreanimated_workaround!(dependencies: computed_dependencies, gutenberg_path: gutenberg_path) unless react_native_version[1] < 71
 
   computed_dependencies.each do |pod_name|
     pod pod_name, podspec: "#{podspec_prefix}/#{pod_name}.#{podspec_extension}"
   end
 end
 
-def gutenberg_pre_install
-  return unless should_use_local_gutenberg
+def apply_rnreanimated_workaround!(dependencies:, gutenberg_path:)
+  # RNReanimated needs React-jsc
+  # Reference: https://github.com/WordPress/gutenberg/pull/51386/commits/9538f8eaf73dfacef17382e1ab7feda40231061f
+  dependencies.push('React-jsc')
 
-  raise "[Gutenberg] Could not find local Gutenberg at given path #{local_gutenberg_path}" unless File.exist?(local_gutenberg_path)
+  # Use a custom RNReanimated version while we coordinate a fix upstream
+  dependencies.delete('RNReanimated')
```

Comment:
For reference, https://github.com/software-mansion/react-native-reanimated/pull/4684